utils/bluez/Makefile

```diff
@@ -70,7 +70,7 @@ define Package/bluez-daemon/conffiles
 /etc/bluetooth/main.conf
 /etc/bluetooth/network.conf
 /etc/bluetooth/input.conf
-/etc/bluetooth/proximity.conf
```

Comment:
why are you removing proximity.conf? are you sure there is no useful information in this conf file (if you upgrade 2 times the router this file will be lost)